src/NotificationTemplate.php

```diff
@@ -470,8 +470,13 @@ public static function processIf($string, $data)
                               $condition_value = __($condition_value);
                         }
 
-                      // Compare data value and condition value
-                        $condition_ok = $condition_value == $data_value;
+                        // Compare data value and condition value
+                        if (strpos($condition_value, "%") !== false) {
+                            $condition_value = str_replace("%", ".*", $condition_value);
+                            $condition_ok = (preg_match("/^" . $condition_value . "$/i", $data_value));
+                        } else {
```

Comment:
Why not directly check if `$condition_value` is a valid regex?
```suggestion
                        $is_regex = preg_match('/^\/.+\//', $condition_value) === 1 && @preg_match($condition_value, '') !== false;
                        if ($is_regex) {
                            $condition_ok = preg_match($is_regex, $data_value) === 1;
                        } else {
```